network/load-balancer/reference-content/configuring-load-balancer.mdx

```diff
@@ -0,0 +1,137 @@
+---
+meta:
+  title: Configuring your Load Balancer
+  description: Learn how to choose different settings and configurations for your Load Balancer depending on your needs
+content:
+  h1: Configuring your Load Balancer
+  paragraph: Learn how to choose different settings and configurations for your Load Balancer depending on your needs
+tags: load-balancer
+categories: 
+  - load-balancer
+  - network
+---
+
+## Load Balancer Overview
+
+Load Balancers allow you to distribute traffic and workload across multiple backend servers. They make it easy to scale your applications while ensuring continuous availability even in the event of heavy traffic. Each Load Balancer that you create is a highly available and fully managed Instance, consisting of one or more frontends and one or more backends. You can configure your Load Balancer according to your needs.
+
+<Lightbox src="scaleway-load-balancer.webp" alt="" />
```

Comment:
I am not sure this is the best picture as it shows a Kapsule LB and a big warning at the top... not the sexiest picture.

Reply:
As above, I think we're seeing different screenshots, can you double check you don't see this?
![image](https://user-images.githubusercontent.com/36301604/200608278-e181f95c-8ebe-4dcc-a45e-0939e2ffa086.png)